deployments/bridges/eth-poa-sub/docker-compose.yml

```
@@ -0,0 +1,101 @@
+# This Compose file should be built using the Rialto and Eth-PoA node
+# compose files. Otherwise it won't work.
+
+version: '3.5'
+services:
+  # We override these nodes to make sure we have the correct chain config for this network.
+  poa-node-arthur: &poa-node
+    volumes:
+      - ./bridges/eth-poa-sub/poa-config:/config
+  poa-node-bertha:
+      <<: *poa-node
+  poa-node-carlos:
+      <<: *poa-node
+
+  # We provide an override for this particular node since this is a public facing
+  # node which we use to connect from things like Polkadot JS Apps.
+  rialto-node-charlie:
+    environment:
+      VIRTUAL_HOST: rialto.bridges.test-installations.parity.io,wss.rialto.brucke.link
+      VIRTUAL_PORT: 9944
+      LETSENCRYPT_HOST: rialto.bridges.test-installations.parity.io,wss.rialto.brucke.link
+      LETSENCRYPT_EMAIL: admin@parity.io
+
+  relay-eth2sub:
```

```suggestion
  relay-headers-poa2rialto:
```
Let's use consistent naming everywhere now.